src/apps/benchmarks/benchmarkPolyfill.c

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <string.h>
```

Comment:
I don't see any code in here using `string.h`

Reply:
`memset` for all of these comments.